test/weights.jl

```diff
@@ -36,6 +36,22 @@ weight_funcs = (weights, aweights, fweights, pweights)
     @test sum(sa, wv) === 7.0
 end
 
+# isequal
+
+@testset "isequal" begin
+
+    x = FrequencyWeights([1; 2; 3], 6)
+
+    y = FrequencyWeights([1; 2; 3], 6) # same values, type and parameters
+    @test isequal(x, y) == true
```

Comment:
FWIW, `== true` is actually redundant here since `@test` will only pass if the returned value is `=== true` (which is even stricter).